modules/govuk_jenkins/files/var/lib/jenkins/groovy_scripts/govuk_jenkinslib.groovy

```diff
@@ -336,10 +336,12 @@ def publishGem(String repository, String branch) {
     sh("gem push ${repository}-${version}.gem")
   }
 
-  def taggedReleaseExists = sh(
-    script: "git ls-remote --exit-code --tags origin v${version}",
-    returnStatus: true
-  ) == 0
+  sshagent(['govuk-ci-ssh-key']) {
+    def taggedReleaseExists = sh(
```

Comment:
`taggedReleaseExists` is now defined in a closure. Is it available outside the closure's scope on line 346? (My guess is no, but I don't know enough Groovy to say for sure.)